src/sentry/integrations/github/issues.py

```diff
@@ -37,13 +37,29 @@ def get_performance_issue_body(self, event: Event) -> str:
         body += f"| **Repeating Spans ({num_repeating_spans})** | {truncatechars(repeating_spans, MAX_CHAR)} |"
         return body
 
-    def get_group_description(self, group: Group, event: Event, **kwargs: Any) -> str:
+    def get_generic_issue_body(self, event: GroupEvent) -> str:
+        body = "|  |  |\n"
+        body += "| ------------- | --------------- |\n"
+
+        important = event.occurrence.important_evidence_display
+        if important:
+            body += f"| **{important.name}** | {truncatechars(important.value, MAX_CHAR)} |\n"
+
+        for evidence in event.occurrence.evidence_display:
+            if evidence.important is False:
+                body += f"| **{evidence.name}** | {truncatechars(evidence.value, MAX_CHAR)} |\n"
```

Comment:
Not sure if it's better, but we could write this like
```
for evidence in sorted(enumerate(event.occurrence.evidence_display), key=lambda evidence: (evidence[1].important, evidence[0]))
    body += f"| **{evidence.name}** | {truncatechars(evidence.value, MAX_CHAR)} |\n"
```

This should sort so that important is first, but the rest of the rows maintain their order. I kind of wonder if this should be something that the occurrence provides?

Reply:
Yeah I was kinda thinking it'd be nice if the occurrence dealt w/ this as I was writing it, since it already provides the important piece of evidence.

Reply:
Ok updated, lmk what you think. Your code was ending up with the important one last, so I read through the [sorted docs](https://docs.python.org/3/howto/sorting.html#sort-stability-and-complex-sorts) and rewrote it, and moved the evidence around a bit to test that it stayed how I expected.

Reply:
You need the `enumerate` to keep the unimportant entries in the same order. I think all you need is `lambda evidence: (not evidence[1].important, evidence[0])`

Reply:
Updated, but the docs I linked say "when multiple records have the same key, their original order is preserved" so wouldn't my implementation have kept them in order too?

Reply:
Ahh, I didn't notice the stable part of the docs. In that case, feel free to do it the way you came up with then, it's easier to read